cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -106,9 +107,9 @@ elif [[ $realHostname == "white"* ]] ; then
 elif [[ $realHostname == "ride"* ]] ; then
   hostnameMatch=ride
   hostnameMatchSystemName=ride
-elif [[ $realHostname == "mutrino"* ]] ; then
+elif [[ $realHostname == "mutrino"* || $HOST == "mutrino"* ]] ; then
   hostnameMatch=mutrino
-  hostnameMatchSystemName=mutrino
+  hostnameMatchSystemName=ats1
```

Comment:
I will add a commit to move this logic out of this if-block and handle it below to add support for both the 'ats1' and 'mutrino' envs on the 'mutrino' cluster.